rllib/execution/metric_ops.py

```diff
@@ -5,15 +5,17 @@
 from ray.util.iter import LocalIterator
 from ray.rllib.evaluation.metrics import collect_episodes, summarize_episodes
 from ray.rllib.execution.common import AGENT_STEPS_SAMPLED_COUNTER, \
-    STEPS_SAMPLED_COUNTER, _get_shared_metrics
+    STEPS_SAMPLED_COUNTER, STEPS_TRAINED_COUNTER, _get_shared_metrics
```

Comment:
For CQL, there is no sampling as the file(s) are pre-loaded at once into the buffer and never touched (read) again.

Reply:
So we'll count training steps instead of sampling steps.